src/main/java/shell/Commands.java

```diff
@@ -77,19 +80,55 @@ static void file(ArrayList<String> args){
                         for(byte e: temp){
                             result += (char)e + " ";
                         }
-                        Shell.print(result);
+                        Shell.println(result);
                     }
                     }
                     catch(IndexOutOfBoundsException e){
-                        Shell.print("Invalid index");
+                        Shell.println("Invalid index");
                     }
                     break;
                 default:
                     Utils.log("Wrong argument");
-                    Shell.print(help);
+                    Shell.println(help);
                     break;
             }
 
         }
     }
+
+    /**
+     * Compiles given program
+     * @param args asm program to compile
+     */
+    static void com(ArrayList<String> args) {
+        if (args.size() == 1) Shell.println("no program file specified");
+        else {
+            final String fileName = args.get(1);
+            final byte[] code = Files.getCleanFile(fileName);
+
+            Assembler assembler = new Assembler();
+            final byte[] exec = assembler.compile(code);
+            if (exec == null) Shell.println("compilation failed");
+            else Files.createFile(fileName.substring(0, fileName.indexOf(".")) + ".exe", exec);
+        }
+    }
+
+    /**
+     * Creates process with given program, name and priority
+     * <p>IN DEVELOPMENT</p>
+     * @param args name of .exe file
+     */
+    static void cp(ArrayList<String> args) {
+        // TODO: implement proper process creation
+        if (args.size() == 1) Shell.println("no exe file specified");
+        else {
+            Utils.log("running program in dev environment");
+            final byte[] exec = Files.getCleanFile(args.get(1));
```

Comment:
```suggestion
            final byte[] exec = Files.getCleanFile(args.get(1));
            if(exec[0] == -1) {
                Shell.println("Program does not exist");
                return;
            }
```

Mała zmiana, bo bez tego próba uruchomiania programu który nie istnieje wysypywała cały system